tests/periph_i2c/tests/01__periph_i2c_base.robot

```diff
@@ -44,3 +44,12 @@ Read Byte After Multiple NACKs Should Succeed
     API Call Should Error       I2C Read Byte  addr=43
     API Call Should Succeed     I2C Read Byte
     API Call Should Succeed     I2C Release
+
+Pins High After Release Should Succeed
+    [Documentation]             Verify recovery of read bytes NACK.
```

Comment:
Seems like a duplicate of line 41 please update the documentation.

Reply:
Done